suite-native/atoms/src/types.ts

```diff
@@ -1,6 +1,6 @@
 export type TestProps = {
     ['data-testid']?: never;
```

Comment:
Can we remove all those variants and use just one? @matejkriz What is the reason for having such many?

Reply:
Good question, though this diff was my mistake, I am reverting it because I just duplicated `'data-test-id'`

Reply:
React Native works only with the `testID` variant and nothing else. This type ensures that the TS shouts to you if you use any other variant than `testID`.

But IMO it is wrong. Even `date-testID` should be defined as `never`.